src/components/OptionListContextProvider.tsx

```diff
@@ -64,22 +64,11 @@ function OptionsListContextProvider({reports, children}: OptionsListProviderProp
             return;
         }
 
-        const lastUpdatedReport = ReportUtils.getLastUpdatedReport();
-
-        if (!lastUpdatedReport) {
-            return;
-        }
```

Comment:
Removing the return early when no existing item need to update allow this `useEffect` to handle adding the new items which should handle in the next useEffect that lead to duplicated new items in the list, and it handled here https://github.com/Expensify/App/issues/43094